packages/confirmation-modal/src/ConfirmationModal.tsx

```diff
@@ -1,154 +1,25 @@
-import { fontFamilies } from '@leafygreen-ui/tokens';
-
-const Mode = {
-  Dark: 'dark',
-  Light: 'light',
-};
-
-type Mode = typeof Mode[keyof typeof Mode];
-
-export const Variant = {
-  Default: ButtonVariant.Primary,
-  Danger: ButtonVariant.Danger,
-} as const;
-
-export type Variant = typeof Variant[keyof typeof Variant];
-
-const titleStyle = css`
-  font-size: 24px;
-  font-weight: 700;
-  line-height: 32px;
-  margin-bottom: 10px;
-  margin-top: 0;
-  color: ${palette.black};
-`;
-
-const baseModalStyle = css`
-  width: 600px;
-  padding: initial;
-  letter-spacing: 0;
-`;
-
-const contentStyle = css`
-  font-family: ${fontFamilies.default};
-  font-size: 13px;
-  line-height: 20px;
-  color: ${palette.black};
-`;
-
-// TODO: Refresh – remove mode logic
-const modeAndVariantContentStyles: Record<Mode, Record<Variant, string>> = {
-  [Mode.Light]: {
-    [Variant.Default]: css`
-      padding: 40px 36px 0px;
-    `,
-    [Variant.Danger]: css`
-      padding: 40px 36px 0px 78px;
-    `,
-  },
-  [Mode.Dark]: {
-    [Variant.Default]: css``,
-    [Variant.Danger]: css``,
-  },
-};
-
-const textEntryInputStyle = css`
-  width: 300px;
-  margin-top: 14px;
-
-  label {
-    margin-bottom: 3px;
-  }
-`;
-
-// TODO: Refresh - remove mode logic
-const buttonStyle = {
-  [Mode.Light]: css`
-    margin: 0 2px;
-
-    &:first-of-type {
-      margin: 0 0 0 5px;
-    }
-
-    &:last-of-type {
-      margin: 0 5px 0 0;
-    }
-  `,
-  [Mode.Dark]: css`
-    margin: 0 2px;
-
-    &:first-of-type {
-      margin: 0 0 0 4px;
-    }
-
-    &:last-of-type {
-      margin: 0 4px 0 0;
-    }
-  `,
-};
-
-const warningIconStyles = css`
-  width: 32px;
-  height: 32px;
-  border-radius: 50%;
-  background: ${palette.red.light3};
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  position: absolute;
-  left: 36px;
-  top: 40px;
-
-  svg {
-    margin-top: -3px;
-  }
-`;
-
-interface ConfirmationModalProps extends ModalProps {
-  /**
-   * Text of header element
-   */
-  title: string;
-
-  /**
-   * The component is shown when the value is set to `true`.
-   */
-  open?: boolean;
-  /**
-   * Callback fired when the primary action button is clicked.
-   */
-  onConfirm?: () => void;
-  /**
-   * Callback fired when the cancel button is clicked.
-   */
-  onCancel?: () => void;
-  className?: string;
-  /**
-   * Text rendered in the primary button. Defaults to `"Confirm"`
-   */
-  buttonText: string;
-  /**
-   * Variant of the modal that represents the type of action handled by the modal.
-   */
-  variant?: Variant;
-  /**
-   * If set, the user will be prompted to type the requiredInputText into an input field
-   */
-  requiredInputText?: string;
-  /**
-   * If `true`, the primary action button will be disabled
-   */
-  submitDisabled?: boolean;
-  darkMode?: boolean;
-}
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import { H3 } from '@leafygreen-ui/typography';
+import {
```

Comment:
ditto on the import * from

Reply:
Gonna merge this in but left a comment about this in the Modal PR